app/src/test/java/ch/hikemate/app/authentication/FirebaseAuthRepositoryTest.kt

```diff
@@ -304,4 +336,48 @@ class FirebaseAuthRepositoryTest {
         verify { mockFirebaseAuth.signOut() }
         verify { onSuccess() }
       }
+
+  @Test
+  fun testDeleteAccount_successful() {
+    runTest(timeout = 5.seconds) {
+      every { mockVoidTask.isSuccessful } returns true
+      every { mockFirebaseAuth.currentUser } returns mockFirebaseUser
+      every { mockFirebaseUser.providerData } returns
+          listOf(mockk { every { providerId } returns EmailAuthProvider.PROVIDER_ID })
+      every { mockDocument.delete() } returns mockVoidTask
+
+      val onSuccess: () -> Unit = mockk(relaxed = true)
+      val mockActivity: Activity = mockk(relaxed = true)
+
+      repository.deleteAccount("password", mockActivity, onSuccess, mockOnError)
```

Comment:
Question: Do we agree that in this setup (i.e. with those mocks), "password" will never be checked for correctness?

Reply:
What do you mean by "will never be checked for correctness? It won't do any request to Firebase for sure. So If i understand your question correctly I would say yes.

Reply:
Then it doesn't check.